src/core/execution/FakeHumanExecution.ts

```
@@ -552,6 +540,52 @@
           }
 
           // TODO: Cities and factories should consider train range limits
+          return w;
+        };
+      case UnitType.SAMLauncher:
+        const structureTiles: TileRef[] = [];
+        for (const unit of this.player.units()) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)